test/unit/org/apache/cassandra/index/sai/cql/BM25Test.java

```diff
@@ -1129,36 +1129,42 @@ private void insertPrimitiveData(int start, int end)
 
     public static void insertCollectionData(SAITester tester)
     {
-        int setsize = 1;
         for (int row = 0; row < DATASET.length; row++)
```

Comment:
I would have probably kept the brace considering the multiline statement, but it is a matter of personal taste. I do not insist on adding anything; I am just expressing my opinion.